src/main/java/core/basesyntax/dao/StorageDao.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface StorageDao {
+    Map<String, Integer> save(String fruit, int quantity);
+
+    Map<String, Integer> getAll();
+
+    int getQuantity(String key);
```

Comment:
The method name `getQuantity` could be more descriptive, such as `getQuantityByFruitName`, to clarify that it retrieves the quantity of a specific fruit.